playground/src/import-url/import-url.scss

```diff
@@ -0,0 +1,47 @@
+@import 'base.scss';
```

Comment:
Beware: this has side-effects as `base` contains other selectors. You probably don't want this.